src/Umbraco.Core/PropertyEditors/ValueConverters/ColorPickerValueConverter.cs

```diff
@@ -1,18 +1,34 @@
-﻿using Umbraco.Core.Models.PublishedContent;
+﻿using System;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Models.PublishedContent;
 
 namespace Umbraco.Core.PropertyEditors.ValueConverters
 {
-    public class ColorPickerValueConverter : PropertyValueConverterBase
+    public class ColorPickerValueConverter : PropertyValueConverterBase, IPropertyValueConverterMeta
```

Comment:
Most of these simple converters shouldn't be using IPropertyValueConverterMeta and use attributes instead.